kudu/pom.xml

```diff
@@ -0,0 +1,299 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.apex</groupId>
+    <artifactId>malhar</artifactId>
+    <version>3.8.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>malhar-kudu</artifactId>
+  <name>Apache Apex Malhar Kudu Support</name>
+  <packaging>jar</packaging>
+
+  <properties>
+    <checkstyle.console>false</checkstyle.console>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.antlr</groupId>
+        <artifactId>antlr4-maven-plugin</artifactId>
+        <version>4.7</version>
+        <executions>
+          <!-- One execution per unit to control output and input accordingly-->
+          <execution>
+            <id>kudusqlantlr4parsergen</id>
+            <configuration>
+              <outputDirectory>${project.basedir}/src/main/java/org/apache/apex/malhar/kudu/sqlparser</outputDirectory>
+            </configuration>
+            <goals>
+              <goal>antlr4</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- The reason we are duplicating the plugin from parent is to take care of checkstyle violations in Antlr4 autogenerated code -->
+      <plugin>
```

Comment:
Removed

Reply:
Because of check style errors emanating from auto generated code, I had to specify the same plugin with additional configuration highly specific to the package structure of Antlr4 Kudu packages. Hence cannot inherit from parent. If we inherit form parent, kudu specific check style config has to be pushed to the parent thereby polluting it.